configure.ac

```diff
@@ -3,7 +3,7 @@
     AC_CONFIG_HEADERS([config.h])
     AC_CONFIG_SRCDIR([src/suricata.c])
     AC_CONFIG_MACRO_DIR(m4)
-    AM_INIT_AUTOMAKE([tar-ustar])
+    AM_INIT_AUTOMAKE([tar-ustar], [subdir-objects])
```

Comment:
Not sure which of the changes it causing this, but I see lots of autogen warnings suddenly:
```
bash autogen.sh 
Found libtoolize
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
libtoolize: 'AC_PROG_RANLIB' is rendered obsolete by 'LT_INIT'
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
autoreconf: configure.ac: tracing
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
autoreconf: configure.ac: adding subdirectory libhtp to autoreconf
autoreconf: Entering directory `libhtp'
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
autoreconf: running: /usr/bin/autoconf --force
autoreconf: running: /usr/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:86: installing './compile'
configure.ac:89: installing './config.guess'
configure.ac:89: installing './config.sub'
configure.ac:7: installing './install-sh'
configure.ac:7: installing './missing'
Makefile.am: installing './INSTALL'
htp/Makefile.am: installing './depcomp'
parallel-tests: installing './test-driver'
autoreconf: Leaving directory `libhtp'
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
libtoolize: 'AC_PROG_RANLIB' is rendered obsolete by 'LT_INIT'
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
configure.ac:466: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required
configure.ac:466: http://www.gnu.org/software/autoconf/manual/autoconf.html#Expanded-Before-Required
../../lib/autoconf/c.m4:667: AC_LANG_COMPILER(C++) is expanded from...
../../lib/autoconf/lang.m4:329: AC_LANG_COMPILER_REQUIRE is expanded from...
../../lib/autoconf/general.m4:2689: AC_LINK_IFELSE is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
../../lib/autoconf/general.m4:2042: AC_CACHE_VAL is expanded from...
m4/libtool.m4:1151: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:6304: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:840: _LT_LANG is expanded from...
m4/libtool.m4:822: LT_LANG is expanded from...
../../lib/m4sugar/m4sh.m4:639: AS_IF is expanded from...
configure.ac:466: the top level
configure.ac:6: warning: AM_INIT_AUTOMAKE: two- and three-arguments forms are deprecated.  For more info, see:
configure.ac:6: http://www.gnu.org/software/automake/manual/automake.html#Modernize-AM_005fINIT_005fAUTOMAKE-invocation
configure.ac:9: installing './compile'
configure.ac:10: installing './config.guess'
configure.ac:10: installing './config.sub'
configure.ac:6: installing './install-sh'
configure.ac:6: installing './missing'
src/Makefile.am:567: warning: source file 'tests/fuzz/fuzz_applayerparserparse.c' is in a subdirectory,
src/Makefile.am:567: but option 'subdir-objects' is disabled
automake: warning: possible forward-incompatibility.
automake: At least a source file is in a subdirectory, but the 'subdir-objects'
automake: automake option hasn't been enabled.  For now, the corresponding output
automake: object file(s) will be placed in the top-level directory.  However,
automake: this behaviour will change in future Automake versions: they will
automake: unconditionally cause object files to be placed in the same subdirectory
automake: of the corresponding sources.
automake: You are advised to start using 'subdir-objects' option throughout your
automake: project, to avoid future incompatibilities.
src/Makefile.am:573: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:573: but option 'subdir-objects' is disabled
src/Makefile.am:554: warning: source file 'tests/fuzz/fuzz_applayerprotodetectgetproto.c' is in a subdirectory,
src/Makefile.am:554: but option 'subdir-objects' is disabled
src/Makefile.am:560: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:560: but option 'subdir-objects' is disabled
src/Makefile.am:592: warning: source file 'tests/fuzz/fuzz_confyamlloadstring.c' is in a subdirectory,
src/Makefile.am:592: but option 'subdir-objects' is disabled
src/Makefile.am:598: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:598: but option 'subdir-objects' is disabled
src/Makefile.am:644: warning: source file 'tests/fuzz/fuzz_decodeder.c' is in a subdirectory,
src/Makefile.am:644: but option 'subdir-objects' is disabled
src/Makefile.am:650: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:650: but option 'subdir-objects' is disabled
src/Makefile.am:605: warning: source file 'tests/fuzz/fuzz_decodepcapfile.c' is in a subdirectory,
src/Makefile.am:605: but option 'subdir-objects' is disabled
src/Makefile.am:611: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:611: but option 'subdir-objects' is disabled
src/Makefile.am:631: warning: source file 'tests/fuzz/fuzz_mimedecparseline.c' is in a subdirectory,
src/Makefile.am:631: but option 'subdir-objects' is disabled
src/Makefile.am:637: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:637: but option 'subdir-objects' is disabled
src/Makefile.am:579: warning: source file 'tests/fuzz/fuzz_siginit.c' is in a subdirectory,
src/Makefile.am:579: but option 'subdir-objects' is disabled
src/Makefile.am:585: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:585: but option 'subdir-objects' is disabled
src/Makefile.am:618: warning: source file 'tests/fuzz/fuzz_sigpcap.c' is in a subdirectory,
src/Makefile.am:618: but option 'subdir-objects' is disabled
src/Makefile.am:624: warning: source file 'tests/fuzz/onefile.c' is in a subdirectory,
src/Makefile.am:624: but option 'subdir-objects' is disabled
src/Makefile.am: installing './depcomp'
autoreconf: Leaving directory `.'
You can now run "./configure" and then "make".
```

Reply:
The warnings come mostly from configure.ac:466 but I did not manage to get rid of them... Help on auto tool is welcome ;-)